streamz/tests/test_kafka.py

```diff
@@ -202,6 +204,7 @@ def test_kafka_batch():
         stream = Stream.from_kafka_batched(TOPIC, ARGS)
         out = stream.sink_to_list()
         stream.start()
+        sleep(2)
```

Comment:
This may cause this test to fail, because default poll_interval is 1 sec and the test would have polled 2 batches. The second batch would be empty list and out[-1][-1] would be null always.

Reply:
I'm not sure about that, I think the time accounts for the initial setting up of the consumer, and no data has been sent yet at this point.

Reply:
Yes, if I had another way to do this without a sleep, I think that would be preferable, but the delay for the topic creation is why it is there.

Reply:
Yeah, you are right! Data hasn't been sent yet.